.eslintrc.json

```diff
@@ -1,14 +1,12 @@
 {
   "parserOptions": {
-    "ecmaVersion": 6,
+    "ecmaVersion": 2017,
     "sourceType": "module",
     "ecmaFeatures": {
       "jsx": true
     }
   },
-  "plugins": [
-    "react"
-  ],
+  "plugins": ["react"],
```

Comment:
just curious but why did you change this to inline?

Reply:
Only one item, can fit in one line

Reply:
It's a prettier preset.  I think 80 characters is the limit for making line breaks.  All the formatting changes were auto-saved by prettier so there is a lot of noise :(

Reply:
gotcha, thanks

Reply:
Does this prettier preset remove use of arrow functions?

Reply:
No it doesn't, but it defaults to a style of `param => { ... }` instead of `(param) => { ... }`.  The methods on the models use a traditional function because I believe those methods will need `this` to point to the model in order to do stuff.

Reply:
For multiple params, it will always be in `(param1, param2) => {}` because javascript requires that.

Reply:
I see, i saw some arrow functions on the other files. i saw one change from () => {} to function() {}.

Reply:
In most cases, we can use arrow functions but there are a few edge cases where we need a `function` declaration in order to access `this` correctly or accessing `arguments` within the function.

Reply:
Still figuring those edge cases out. I remember some guys running into issues at work with 'this' binding in es5.